zebrad/src/commands.rs

```diff
@@ -5,59 +5,47 @@ mod download;
 mod generate;
 mod start;
 mod tip_height;
-mod version;
 
 use self::ZebradCmd::*;
 use self::{
     copy_state::CopyStateCmd, download::DownloadCmd, generate::GenerateCmd,
-    tip_height::TipHeightCmd, version::VersionCmd,
+    tip_height::TipHeightCmd,
 };
 
 pub use self::start::StartCmd;
 
 use crate::config::ZebradConfig;
 
-use abscissa_core::{
-    config::Override, Command, Configurable, FrameworkError, Help, Options, Runnable,
-};
+use abscissa_core::{config::Override, Command, Configurable, FrameworkError, Runnable};
+use clap::Parser;
+use std::ffi::OsString;
 use std::path::PathBuf;
 
 /// Zebrad Configuration Filename
 pub const CONFIG_FILE: &str = "zebrad.toml";
 
 /// Zebrad Subcommands
-#[derive(Command, Debug, Options)]
+#[derive(Command, Debug, clap::Subcommand)]
 pub enum ZebradCmd {
     /// The `copy-state` subcommand, used to debug cached chain state
     // TODO: hide this command from users in release builds (#3279)
-    #[options(help = "copy cached chain state (debug only)")]
     CopyState(CopyStateCmd),
 
-    /// The `download` subcommand
-    #[options(help = "pre-download required parameter files")]
+    // The `download` subcommand
+    /// Pre-download required parameter files
     Download(DownloadCmd),
 
-    /// The `generate` subcommand
-    #[options(help = "generate a skeleton configuration")]
+    // The `generate` subcommand
+    /// Generate a skeleton configuration
```

Comment:
"Skeleton" is an obscure term for this

```suggestion
    /// Generate a default `zebrad.toml` configuration
```